CONTRIBUTING.md

```diff
@@ -42,11 +42,10 @@ This allows models to be inherited. See the section on _Inherited Callback Queue
 [`ActiveRecord::Callbacks`](http://api.rubyonrails.org/classes/ActiveRecord/Callbacks.html#module-ActiveRecord::Callbacks-label-Inheritable+callback+queues).
 
 ## Views
-Remember to specify page breadcrumbs (using the `add_breadcrumb` helper in controllers. Page titles
-will be generated by reversing breadcrumbs, so the name of current resource will always be clearly
-present at the start of the title (for users to quickly switch to the tab when many tabs are open).
-This will also allow users to jump to a page by the location in the module. The Coursemology
-branding is automatically appended to all page titles.
+Page titles will be generated by reversing breadcrumbs, so the name of current resource will always
+be clearly present at the start of the title (for users to quickly switch to the tab when many tabs
+are open). This will also allow users to jump to a page by the location in the module. The
+Coursemology branding is automatically appended to all page titles.
```

Comment:
Say that programmers can still manually override the page title if necessary, and how to do so.